packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2988,60 +2981,67 @@ export class ContainerRuntime
 			clientId: message.clientId,
 			content: envelope.contents.content,
 			type: envelope.contents.type,
+			targetClientId: message.targetClientId,
 		};
 
-		// Only collect signal telemetry for messages sent by the current client.
-		if (message.clientId === this.clientId && this.connected) {
+		// Only collect signal telemetry for broadcast messages sent by the current client.
+		if (
+			message.clientId === this.clientId &&
+			this.connected &&
+			message.targetClientId === undefined &&
+			envelope.clientBroadcastSignalSequenceNumber !== undefined
+		) {
```

Comment:
Do we need both of these checks?
Getting the feeling that only clientBroadcastSignalSequenceNumber matters.